modules/add_credit/tests/src/FunctionalJavascript/AddCreditCustomAmountTest.php

```diff
@@ -176,6 +167,95 @@ public function testUnitPriceValidation(float $minimum = NULL, float $maximum =
     $this->assertSession()->pageTextContains($message);
   }
 
+  /**
+   * Tests adding a custom amount and checking out.
+   */
+  public function testCustomAmountPriceCheckout() {
```

Comment:
Oh, man  this test alone now takes 7 minutes on my local, Maybe we should add a todo to make this test more efficient. It looks like `@dataProvider` installs a copy of drupal for every each item.

Reply:
Hmm are you sure it's this test? This one does not use a `dataProvider`.

Reply:
I was refering to `AddCreditCustomAmountTest.php`.